### PR TITLE
Convert Relative Paths to Full Paths In Schema

### PR DESCRIPTION
**Pull request by GavinZZ**
Add schema change

Use HTML extension for docs

Format file

Format file

### Issue #, if available

#2662 

### Checklist

- [ ] Adheres to the [development guidelines](https://github.com/aws/serverless-application-model/blob/develop/DEVELOPMENT_GUIDE.md#development-guidelines)
- [ ] Add/update [unit tests](https://github.com/aws/serverless-application-model/blob/develop/DEVELOPMENT_GUIDE.md#unit-testing-with-multiple-python-versions) using:
    - [ ] Correct values
    - [ ] Bad/wrong values (None, empty, wrong type, length, etc.)
    - [ ] [Intrinsic Functions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference.html)
- [ ] Add/update [integration tests](https://github.com/aws/serverless-application-model/blob/develop/INTEGRATION_TESTS.md)
- [ ] Update documentation
- [ ] Verify transformed template deploys and application functions as expected
- [ ] Do these changes include any template validations?
    - [ ] Did the newly validated properties support intrinsics prior to adding the validations? (If unsure, please review [Intrinsic Functions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference.html) before proceeding).
        - [ ] Does the pull request ensure that intrinsics remain functional with the new validations?

### Examples?

Please reach out in the comments, if you want to add an example. Examples will be 
added to `sam init` through https://github.com/awslabs/aws-sam-cli-app-templates/

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by hoffa — APPROVED** [review]

File: samtranslator/schema/docs.json [hunk 1]
```diff
@@ -3,19 +3,19 @@
...
+      "Authorizers": "The authorizer used to control access to your API Gateway API\\.  \nFor more information, see [Controlling access to API Gateway APIs](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-controlling-access-to-apis.html)\\.  \n*Type*: [CognitoAuthorizer](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-api-cognitoauthorizer.html) \\| [LambdaTokenAuthorizer](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-api-lambdatokenauthorizer.html) \\| [LambdaRequestAuthorizer](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-api-lambdarequestauthorizer.html)  \n*Required*: No  \n*Default*: None  \n*AWS CloudFormation compatibility*: This property is unique to AWS SAM and doesn't have an AWS CloudFormation equivalent\\.  \n*Additional notes*: SAM adds the Authorizers to the OpenApi definition of an Api\\.",
```
Comment (hoffa) [inline samtranslator/schema/docs.json:6]: From elsewhere in the file:

```
[UsagePlan property is specified](sam-specification-generated-resources-api.md#sam-specification-generated-resources-api-usage-plan)
```

Comment (GavinZZ) [reply samtranslator/schema/docs.json:6]: Fixed

**Review by xazhao — APPROVED** [review]

File: bin/parse_docs.py [hunk 2]
```diff
@@ -36,6 +36,21 @@ def remove_first_line(s: str) -> str:
     return s.split("\n", 1)[1]
 
 
+def convert_to_full_path(description: str) -> str:
+    pattern = re.compile("\(([#\.a-zA-Z0-9_-]+)\)")
```
Comment (aahung) [inline bin/parse_docs.py:40]: nit: we can move it to constant to avoid repeat compile

Comment (aahung) [reply bin/parse_docs.py:40]: can we give it a name to explain what we are trying to match?

**Review by aahung — APPROVED** [review]

**Pull request merged**